packages/ui/src/pages/SignIn/IdentifierSignInForm/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { SignInIdentifier } from '@logto/schemas';
+import type { SignIn } from '@logto/schemas';
+import classNames from 'classnames';
+import { useState, useCallback, useMemo } from 'react';
+import { useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+
+import Button from '@/components/Button';
+import ErrorMessage from '@/components/ErrorMessage';
+import type { IdentifierInputType } from '@/components/InputFields';
+import { SmartInputField } from '@/components/InputFields';
+import TermsOfUse from '@/containers/TermsOfUse';
+import useTerms from '@/hooks/use-terms';
+import { identifierErrorWatcher, validateIdentifierField } from '@/utils/form';
+
+import * as styles from './index.module.scss';
+import useOnSubmit from './use-on-submit';
+
+type Props = {
+  className?: string;
+  // eslint-disable-next-line react/boolean-prop-naming
+  autoFocus?: boolean;
+  signInMethods: SignIn['methods'];
+};
+
+type FormState = {
+  identifier: string;
+};
+
+const IdentifierSignInForm = ({ className, autoFocus, signInMethods }: Props) => {
```

Comment:
It seems that "IdentifierSignIn" and "PasswordSignIn" are the same thing?

Reply:
With PasswordInput Field and without.